doc/manifest/schema/1.0.0/installer.md

```diff
@@ -0,0 +1,397 @@
+</details>
+
+### Silent
+<details>
+ <summary>Silent is the value that should be passed to the installer when user chooses a silent or quiet install</summary>
+
+ #### Optional Field
+ This key represents switches passed to the installer to provide a silent install experience.
+
+ >Note: When the Windows Package Manager installs a package using the "silent" install mode, any custom switches will also be passed to the installer. If a user applies override switches via command line via the Windows Package Manager, none of the switches from the manifest will be passed to the installer.
+ 
+ >Note: This key may be present in the root of the manifest as the default value for all installer nodes. This key may also be present in an individual installer node as well. If this key is in the manifest root and in an installer node, the value in the installer node will apply.
+</details>
+
+### SilentWithProgress
+<details>
+ <summary>SilentWithProgress is the value that should be passed to the installer when user chooses a non-interactive install</summary>
+
+ #### Optional Field
+ This key represents switches passed to the installer to provide a silent with progress install experience. This is intended to allow a progress indication to the user, and the indication may come from an installer UI dialogue, but it must not require user interaction to complete. The Windows Package Manager currently defaults to this install experience.
+
+ >Note: When the Windows Package Manager installs a package using the "silent with progress" install mode, any custom switches will also be passed to the installer. If a user applies override switches via command line via the Windows Package Manager, none of the switches from the manifest will be passed to the installer.
+</details>
+
+### Interactive
+<details>
+ <summary>Interactive is the value that should be passed to the installer when user chooses an interactive install</summary>
+
+ #### Optional Field
+ This key represents switches passed to the installer to provide an interactive install experience. This is intended to allow a user to interact with the installer.
+
+ >Note: When the Windows Package Manager installs a package using the "interactive" install mode, any custom switches will also be passed to the installer. If a user applies override switches via command line via the Windows Package Manager, none of the switches from the manifest will be passed to the installer.
+
+</details>
+
+### InstallLocation
+<details>
+ <summary>InstallLocation is the value passed to the installer for custom install location. </summary>
+
+ #### Optional Field
```

Comment:
We do not have many samples of the command line with these options.  I wonder if we need to add them.